desktop/libs/notebook/src/notebook/connectors/jdbc.py

```diff
@@ -45,7 +45,10 @@ def decorator(*args, **kwargs):
     except Exception as e:
       message = force_unicode(smart_str(e))
       if 'error occurred while trying to connect to the Java server' in message:
-        raise QueryError, _('%s: is the DB Proxy server running?') % message, sys.exc_info()[2]
+        if sys.version_info[0] > 2:
+          raise QueryError(_('%s: is the DB Proxy server running?') % message).with_traceback(sys.exc_info()[2])
+        else:
+          raise QueryError, _('%s: is the DB Proxy server running?') % message, sys.exc_info()[2]
       elif 'Access denied' in message:
         raise AuthenticationRequired, '', sys.exc_info()[2]
       else:
```

Comment:
the same change need to be added here too : https://github.com/cloudera/hue/blob/64476fb530e6d16fa8869c4ec144f807095140e3/desktop/libs/notebook/src/notebook/connectors/jdbc.py#L55

```
     else:
        raise QueryError, message, sys.exc_info()[2]
```